plugins/__init__.py

```diff
@@ -250,7 +253,7 @@ def decorator(func):
                             "event listener call). It was not added to the list of events.")
             return func
 
-        if self and not bot and client.user.bot:
+        if self and not bot and discord.User.bot:
```

Comment:
remove as user accounts are no longer supported

```suggestion
        if self and not bot:
```